share/search/daemon.py

```diff
@@ -82,7 +79,7 @@ def stop_daemonthreads(self, *, wait=False):
 
 
 class KombuMessageConsumer(ConsumerMixin):
-    PREFETCH_COUNT = 7500
+    PREFETCH_COUNT = settings.ELASTICSEARCH['CHUNK_SIZE']
```

Comment:
It looks like this is currently set to 250 on prod. Should it be higher?

Reply:
might be the default should be lower! maybe worth experimenting during a backfill

Reply:
We can keep it at 250 and see how it behaves. If the issue was that rabbit was timing out prefetched items on partial-chunk acks, having them be equal sizes should mitigate for most sensible values.